tests/config_test.go

```diff
@@ -574,6 +578,242 @@ var _ = Describe("[rfe_id:899][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 			}, 200*time.Second)).To(Succeed())
 		})
 	})
+
+	Context("With a virtiofs filesystem defined", func() {
+		BeforeEach(func() {
+			checks.SkipTestIfNoFeatureGate(virtconfig.VirtIOFSGate)
+		})
+
+		Context("With a single ConfigMap volume", func() {
+			var (
+				configMapName string
+				configMapPath string
+			)
+
+			BeforeEach(func() {
+				// We use the ConfigMap name as mount `tag` for qemu, but the `tag` property must be 36 bytes or less
+				configMapName = "configmap-" + uuid.NewRandom().String()[:6]
+				configMapPath = config.GetConfigMapSourcePath(configMapName)
+
+				data := map[string]string{
+					"option1": "value1",
+					"option2": "value2",
+					"option3": "value3",
+				}
+				tests.CreateConfigMap(configMapName, testsuite.NamespacePrivileged, data)
+			})
+
+			AfterEach(func() {
+				tests.DeleteConfigMap(configMapName, testsuite.NamespacePrivileged)
```

Comment:
Is this necessary? Isn't the configmap automatically cleaned up at the end of the test?

Reply:
After testing it, It seems it's unnecessary, so I'll delete it